scripts/release.sh

```diff
@@ -116,25 +116,23 @@ clientdir="$(p4 g4d -- "${citcclient?}")"
 
 cd "${clientdir}"
 
-blaze build java/com/google/cloud/java/tools:ReleaseRapidProject
-if [[ "${SUFFIX}" = "bom" ]]; then
-  blaze-bin/java/com/google/cloud/java/tools/ReleaseRapidProject \
-      --project_name=cloud-java-tools-cloud-opensource-java-bom-kokoro-release \
-      --version=${VERSION} --committish_suffix=${SUFFIX}
+RELEASE_RAPID_PROJECT=java/com/google/cloud/java/tools:ReleaseRapidProject
+blaze build "${RELEASE_RAPID_PROJECT}"
+
+release_rapid_project() {
+  local project="$1"
+  "blaze-bin/${RELEASE_RAPID_PROJECT/://}" \
+      --project_name="cloud-java-tools-cloud-opensource-java-${project}-release" \
+      --version="${VERSION}" --committish_suffix="${SUFFIX}"
+}
```

Comment:
Probably should have a blank line after this function declaration.

Reply:
Added a blank line.